src/app/about/officer-card.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from "next/image";
+import styles from "./OfficerCard.module.css";
+
+type SocialLink = {
+  name: string;
+  url: string;
+  icon: string;
+};
+
+export type OfficerCardProperties = {
+  position: string;
+  image: string;
+  alt: string;
```

Comment:
[Comment/Suggestion]: It's unusual to see an `alt` text property for a generic component like this (especially given it's set to an empty string when used below). I would suggest computing something like `Profile picture of ${name}.` instead, I think that's good enough for our needs.